bin/service-status.py

```diff
@@ -83,20 +82,17 @@ def get_config():
 
 
 def fetch(url, is_json=True):
-    resp = requests.get(url, timeout=5)
-    if resp.status_code != 200:
-        print(url)
-        print(f"{resp.status_code}, {resp.content}")
-        raise Exception("Bad return code")
+    """Fetch data from a url
+
+    This raises URLError on HTTP request errors. It also raises JSONDecode
+    errors if it's not valid JSON.
+
+    """
+    fp = urlopen(url, timeout=5)
```

Comment:
this needs to suppress S310 or perhaps in pyproject.toml
```suggestion
    fp = urlopen(url, timeout=5)  # noqa: S310
```